### ISSUE TITLE
Migrate map and monitor and scoreboard to json

### ISSUE THREAD

**Issue by cyberpwnn** [VolmitSoftware/React#310]
The monitor map and scoreboard would be Parsed via json instead of hard coded to allow additional monitoring types 

### Monitor (title)
```json 
[
    {
        "head": "tps",
        "title": "Tick",
        "fields": [
            "tick",
            "cpu"
        ]
    },
    {
        "head": "mem",
        "title": "Memory",
        "fields": [
            "mahs",
            "memtotals"
        ]
    },
    {
        "head": "chk",
        "title": "Chunks",
        "fields": ["chks"]
    },
    {
        "head": "react_time",
        "title": "Test?",
        "fields": ["react_task_time"]
    },
    {
        "head": "ent",
        "title": "Entities",
        "fields": [
            "entliv",
            "entdrop",
            "enttile"
        ]
    }
]
```

**Comment by cyberpwnn** [comment]
Monitor migrated

**Comment by cyberpwnn** [comment]
Implemented in 6.4.1
